voila/execute.py

```diff
@@ -107,6 +107,14 @@ def preprocess(self, nb, resources, km=None):
             result = (nb, resources)
         return result
 
+    def preprocess_cell(self, cell, resources, cell_index, store_history=True):
+        try:
+            result = super(VoilaExecutePreprocessor, self).preprocess_cell(cell, resources, cell_index, store_history)
+        except CellExecutionError as e:
+            self.log.error(e)
+            result = [cell]
+        return result
+
```

Comment:
Why is this needed?

Reply:
To mimic the `VoilaExecutePreprocessor.preprocess` behavior. I change this behavior a bit in a separate PR concerning error outputs BTW.

Reply:
The `try except` prevents a `505` server error when the Notebook contains an error